cmd/kubeless/function/function.go

```diff
@@ -44,11 +47,43 @@ var FunctionCmd = &cobra.Command{
 	Short: "function specific operations",
 	Long:  `function command allows user to list, deploy, edit, delete functions running on Kubeless`,
 	Run: func(cmd *cobra.Command, args []string) {
+		cli := utils.GetClientOutOfCluster()
+		controllerNamespace := os.Getenv("KUBELESS_NAMESPACE")
+		kubelessConfig := os.Getenv("KUBELESS_CONFIG")
+
+		if len(controllerNamespace) == 0 {
+			controllerNamespace = "kubeless"
+		}
+
+		if len(kubelessConfig) == 0 {
+			kubelessConfig = "kubeless-config"
+		}
+		config, err := cli.CoreV1().ConfigMaps(controllerNamespace).Get(kubelessConfig, metav1.GetOptions{})
+		if err != nil {
+			logrus.Fatalf("Unable to read the configmap: %v", err)
+		}
+
+		var lr = langruntime.New(config)
+		lr.ReadConfigMap()
+
+		getServerConfig, err := cmd.Flags().GetBool("get-server-config")
+		if err != nil {
+			logrus.Fatal(err)
+		}
+
+		if getServerConfig == true {
+			logrus.Info("Current Server Config:")
+			logrus.Infof("Supported Runtimes are: %s",
+				strings.Join(lr.GetRuntimes(), ", "))
+			return
+		}
+
 		cmd.Help()
 	},
 }
 
 func init() {
+	FunctionCmd.Flags().Bool("get-server-config", false, "Get the current server configuration")
```

Comment:
I would rather make this a command, just mimicking `gcloud container get-server-config`. Also making this a flag may be confusing for the the rest of commands.